crowdin.yml

```diff
@@ -1393,6 +1393,23 @@ files:
         sr: sr_RS
         ku: kmr_TR
         kmr: kmr
+  - source: /knowledge_panels/obf/ingredients/world-en.yml
+    translation: /knowledge_panels/obf/ingredients/world-%two_letters_code%.yml
```

Comment:
There also wild card are supported by crowdin, why don't we use them ?

```yaml
  - source: "/knowlegde_panels/**/world-en.yml"
    target: "%original_path%/world-%two_letters_code%.yml"
```
should work for any knowledge_panels and avoid a lot of line and potential misses